Objects/listobject.c

```diff
@@ -2145,12 +2145,21 @@ list.sort
     key as keyfunc: object = None
     reverse: bool(accept={int}) = False
 
-Stable sort *IN PLACE*.
+Sort the list in ascending order.
+
+The sort is in-place (i.e. the list itself is modified) and stable (i.e. the
+order of two equal elements is maintained).
```

Comment:
Change to the following with more information
" In any case, the sort is stable -- items with equal values or equal key function values keep their order."
and put it last.

This makes it clear that a descending sort is *not* an ascending sort reversed.
The latter is an easy and 'obvious' possible implementation, but would make descending sorts anti-stable.
I have sometimes wondered if 'stable' only applied to the default 'ascending' and just checked to be sure that it is not.  Others might wonder too.  I also checked that this remains true with key values.

Reply:
I prefer the OP's short version.  More discussion of stability belongs in a FAQ or HOWTO.

Reply:
It is good enough.